spec/features/dashboard_spec.rb

```diff
@@ -137,9 +137,9 @@
     it "displays a list of upcoming holidays" do
       within("#holidays") do
         expect(page).to have_content("Upcoming Holidays")
-        expect(page).to have_content("Columbus Day")
         expect(page).to have_content("Veterans Day")
         expect(page).to have_content("Thanksgiving Day")
+        expect(page).to have_content("Christmas Day")
       end
     end
 end
```

Comment:
We can make this perpetually dynamic instead, so it doesn't fail again after Veterans Day. It can be a fix for later.